tests/python/relay/test_pass_partition_graph.py

```diff
@@ -856,6 +857,128 @@ def expected():
     partitioned = transform.PartitionGraph()(mod)
     assert tvm.ir.structural_equal(partitioned, ref_mod, map_free_vars=True)
 
+
+def test_dnnl_fuse():
+    def make_pattern(with_bias=True):
+        data = relay.var("data", relay.TensorType((1, 3, 224, 224), "float32"))
+        weight = relay.var("weight")
+        bias = relay.var("bias")
+        conv = relay.nn.conv2d(data=data, weight=weight, kernel_size=(3, 3),
+                               channels=8, padding=(1, 1))
+        if with_bias:
+            conv_out = relay.add(conv, bias)
```

Comment:
Should this be ```relay.nn.bias_add```? On my side, I had ```nn.bias_add``` after conv2d and noticed that the GetRootCall would fail unless I changed ```add``` to ```nn.bias_add```.

Reply:
No, since I use this pattern to detect conv + add + relu that come from decomposing the batch norm.  `SimplyfyInference` generates `relay.add`, see https://github.com/apache/incubator-tvm/blob/14ba49c60c49474a564f990363de9d114c9b019b/src/relay/transforms/simplify_inference.cc#L54

We can have both conv2d + add + relu and conv2d + bias_add + relu patterns in the table.

Reply:
Understood, thank you.